src/Stripe.net/Services/SetupIntents/SetupIntentConfirmOptions.cs

```diff
@@ -4,13 +4,6 @@ namespace Stripe
 
     public class SetupIntentConfirmOptions : BaseOptions
     {
-        /// <summary>
-        /// The client secret of the SetupIntent. Required if a publishable key is used to
-        /// confirm the setup intent.
-        /// </summary>
-        [JsonProperty("client_secret")]
-        public string ClientSecret { get; set; }
```

Comment:
This needs to stay here. This is how you use stripe-dotnet to confirm a PI or SI in a mobile app right?

Reply:
As in using stripe-dotnet in the mobile app with xamarin or something?

Reply:
Yes

Reply:
ack will add with an override :)

Reply:
It's still removed incorrect right?

Reply:
woops yeah didn't push the right thing. should be fixed now.